netjson-monitoring.lua

```diff
@@ -119,6 +119,27 @@ load_average = {tonumber(loadavg_output[1]),
                 tonumber(loadavg_output[2]),
                 tonumber(loadavg_output[3])}
 
+function parse_disk_usage()
+    file = io.popen('df')
+    disk_usage_info = {}
+    for line in file:lines() do
+        if line:sub(1, 10) ~= 'Filesystem' then
+            filesystem, memory, used, available, percent, location = line:match("(%S+)%s+(%S+)%s+(%S+)%s+(%S+)%s+(%S+)%s+(%S+)")
+            table.insert(disk_usage_info, {
+                name = filesystem,
+                -- in 1k blocks
+                size = memory,
+                used_space = used,
+                available_space = available,
+                percent_used = percent,
```

Comment:
Can you remove the trailing % sign?

Reply:
Sure, took care of it on the openwisp instance side but if you say I will make those changes here